master/reference/advanced/etcd-rbac/kubernetes-advanced.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Advanced Kubernetes set ups utilizing etcd RBAC
+---
+
+When using etcd with RBAC, all components that access etcd must be configured
+with the proper certificates.  This document describes advanced configuration
+options of individual component that can be configured and offers some help
```

Comment:
advanced configuration options sounds like we are describing more than we actually are -  aren't we just describing how to limitaccess to the etcd datastore on a component-by-component basis.